docs/alerts.rst

```diff
@@ -0,0 +1,242 @@
+
+.. code-block:: python
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> from cbc_sdk.platform import WatchlistAlert, Process
+    >>> api = CBCloudAPI(profile='sample')
+    >>> alerts = api.select(WatchlistAlert)[:10]
+    >>> for alert in alerts:
+    ...     process = api.select(Process).where(process_guid=alert.original_document['process_guid']).first()
+    ...     print(process.get_details())
+    {'alert_category': ['OBSERVED', 'THREAT'], 'alert_id': ['06eca427-1e64-424<truncated>..}
+    {'alert_category': ['OBSERVED', 'THREAT'], 'alert_id': ['2307bf6e-fd39-4b6<truncated>..}
+    ...
+
+Get Process Events
+^^^^^^^^^^^^^^^^^^
+
+We could also fetch every event which corresponds with our Process, we can do so by calling ``process.events()``.
+
+.. note::
+    Since calling the events could be really intensive task in the example below we are fetching just the first ``10``
+    events. Be careful when calling ``all()``.
+
+
+.. code-block:: python
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> from cbc_sdk.platform import WatchlistAlert, Process
+    >>> api = CBCloudAPI(profile='sample')
+    >>> alert = api.select(WatchlistAlert).first()
+    >>> process = api.select(Process).where(process_guid=alert.original_document['process_guid']).first()
+    >>> events = process.events()[:10]
+    >>> print(events[0].original_document['event_description']) # Note that I've striped the `<share>` and `<link>` tags which are also available in the response.
+    'The application c:\\program files (x86)\\google\\chrome\\application\\chrome.exe attempted to modify the memory of "c:\\program files (x86)\\google\\chrome\\application\\chrome.exe", by calling the function "NtWriteVirtualMemory". The operation was successful.'
+    ...
+
+
+Device Control Alerts
+---------------------
+
+The Device Control Alerts are explained in the :doc:`device-control` guide.
+
+
+Migrating from Notifications to Alerts
+--------------------------------------
+
+The notifications are working on a subscription based principle and they require a ``SIEM`` key of authentication.
+With that key you are subscribing to a certain criteria of alerts, note that only CB Analytics and Watchlist alerts
+can be retrieved from the notifications API.
+
+Please referer to `the official notes <https://developer.carbonblack.com/reference/carbon-black-cloud/cb-defense/latest/rest-api/#get-notifications>`_ in the Carbon Black's API website.
+
+.. image:: _static/cbc_platform_notification_edit.png
+   :alt: Editing a notification in the CBC Platform
+   :align: center
+
+Those settings shown in the screenshot can be replicated with the following code:
+
+
+.. code-block:: python
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> from cbc_sdk.platform import BaseAlert
+    >>> from solrq import Q
+    >>> api = CBCloudAPI(profile='sample')
+    >>> alerts = api.select(BaseAlert).where((Q(category='INFO') | Q(category='THREAT')), policy_name='Standard').set_minimum_severity(7)[:5]
```

Comment:
We haven't been using the solrq style in many places. I would consider using this as a `set_categories` and where clause query string instead or just a query string.